chsdi/models/vector/stopo.py

```diff
@@ -1067,26 +1113,9 @@ class GeolGeocoverMetadata(Base, Vector):
 register('ch.swisstopo.geologie-geocover.metadata', GeolGeocoverMetadata)
 
 
-class GeolGenKarteGGK200(Base, Vector):
-    __tablename__ = 'kv_ggk_pk'
-    __table_args__ = ({'schema': 'geol', 'autoload': False})
-    __template__ = 'templates/htmlpopup/generalkarte_ggk200.mako'
-    __bodId__ = 'ch.swisstopo.geologie-generalkarte-ggk200'
-    __label__ = 'titel'
-    id = Column('gid', Integer, primary_key=True)
-    nr = Column('nr', Integer)
-    titel = Column('titel', Text)
-    url_legend = Column('url_legend', Text)
-    the_geom = Column(Geometry(geometry_type='GEOMETRY',
-                               dimension=2, srid=21781))
-
-register('ch.swisstopo.geologie-generalkarte-ggk200', GeolGenKarteGGK200)
-
-
 class GeolGenKarteGGK200Meta(Base, Vector):
     __tablename__ = 'kv_ggk_pk'
     __table_args__ = ({'schema': 'geol', 'autoload': False, 'extend_existing': True})
-    __template__ = 'templates/htmlpopup/generalkarte_ggk200.metadata.mako'
```

Comment:
removing this line makes jenkins to fail.
You should either keep it or make the class a child of ShopProductGroupClass